src/components/DetailsPreview/DetailsPreview.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ArtifactsPreview from '../ArtifactsPreview/ArtifactsPreview'
+
+import { ReactComponent as Popout } from '../../images/popout.svg'
+
+const DetailsPreview = ({ artifact, handlePreview }) => {
+  if (artifact.extra_data) {
+    artifact.target_path.path = Object.values(
+      artifact.extra_data
+    ).find(dataItem => dataItem.match(/html/))
+  }
```

Comment:
There is a bug here. If path starts with a schema it should not be included in `target_path.path`.
For example, you can add this line here:
```js
artifact.target_path.path = artifact.target_path.path.replace(/^(.*)+:\/\//, '')
```
I found it by the real-word example in /projects/gitops-project/artifacts/train_model/preview:
```js
{
  confusion: "v3io:///users/admin/pipe/920dd33c-147c-4d6f-8408-e0adfc7a8c0b/2/plots/train/confusion.html"
}
```
This crashes because back-end responds with 500 Internal Server Error when `path` URL search param includes the schema.